docs/meshstack.azure.index.md

```diff
@@ -188,7 +188,7 @@ If this call **does not** return a usable ID then you can try an alternative way
 5. List your Blueprint Service Principal ID via
 
     ```text
-    GET https://graph.microsoft.com/beta/<AAD_TENANT_ID>/servicePrincipals?$filter=appId eq 'f71766dc-90d9-4b7d-bd9d-4499c4331c3f'
+    GET https://graph.microsoft.com/beta/&lt;AAD_TENANT_ID&gt;/servicePrincipals?$filter=appId eq 'f71766dc-90d9-4b7d-bd9d-4499c4331c3f'
```

Comment:
d: should we still describe `#### Finding the Azure Blueprints App Id` altnerative via Graph API in our public docs? Ie what are cases in which PS `Get-AzureRmADServicePrincipal` does not work but graph API does? 

Maybe we should separate this out into a `Guide: Azure Blueprints App Id` page and link it from here so that the high-level platform integration instructions are still followable